storybook/stories/CenterView/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+
+const CenterView = ({ children }: any) => {
```

Comment:
[comment]
ここはReactNodeの方がいいかもですね
```suggestion
const CenterView = ({ children }: ReactNode) => {
```